rhel8/Dockerfile

```diff
@@ -82,6 +82,7 @@ RUN if [ "$DRIVER_TYPE" != "vgpu" ] && [ "$TARGETARCH" != "arm64" ]; then \
     dnf install -y ${fmPackage} ${nscqPackage}; fi
 
 RUN if [ "$DRIVER_TYPE" != "vgpu" ] && [ "$DRIVER_BRANCH" -ge "550" ]; then \
+    dnf module enable -y nvidia-driver:${DRIVER_BRANCH}-dkms && \
```

Comment:
Question -- should we not do this once when setting up the CUDA repository? We are duplicating this line here and before we install FM / nscq.

Reply:
Yeah, I think that makes sense

Reply:
Please check now